demo/demo-pojo/pojo-server/src/main/resources/microservice.yaml

```diff
@@ -24,7 +24,7 @@ servicecomb:
     registry:
       address: http://127.0.0.1:30100
   rest:
-    address: 0.0.0.0:8080
+    address: 0.0.0.0:8080?protocol=h2
```

Comment:
It's better to use protocol=http2, because
1. protocol=http2 + sslEnabled=true means h2
2. protocol=http2 + sslEnabled=false means h2c